src/common/paddle.ts

```diff
@@ -54,3 +63,125 @@ export const getPriceFromPaddleItem = (
 
   return priceAmount / 100;
 };
+
+export enum ProductPurchaseType {
+  Plus = 'plus',
+  Core = 'core',
+}
+
+export const getProductPurchaseType = ({
+  id,
+}: {
+  id: string;
+}): ProductPurchaseType => {
+  if (!remoteConfig.vars.coreProductId) {
+    throw new Error('Core product id is not set');
+  }
+
+  switch (id) {
+    case remoteConfig.vars.coreProductId:
+      return ProductPurchaseType.Core;
+    default:
+      return ProductPurchaseType.Plus;
+  }
+};
+
+const paddleNotificationCustomDataSchema = z.object(
```

Comment:
Should we consider moving the schemas to a separate file?

Reply:
Yes, but I also expect a lot more usage in the future so maybe each `common` can have its own schemas like this. I'll think about it 👍